src/geolocation/api.rs

```diff
@@ -0,0 +1,20 @@
+pub const MAXMIND_CITY_URI: &str = "https://geoip.maxmind.com/geoip/v2.1/city";
+
+pub struct MaxMindAPIKey(String, String);
```

Comment:
Since both fields are of the same type, it would make sense to name those in order to differentiate between them.

Reply:
Access to both fields are private (consumers can get the username/password using the `.username()` and `.password()` methods), but perhaps a named struct would be more clear, yes.

Reply:
Fixed in 2d15a5c